### PR TITLE
Support marks

### PR DESCRIPTION
**Pull request by ammgws**
As per title

### CODE REVIEW

**Comment by ammgws** [comment]
By the way I've tested this and it works fine 👌

File: src/reply.rs [hunk 1]
```diff
@@ -215,6 +215,7 @@ pub struct Node {
     pub pid: Option<i32>,
     pub window: Option<i32>,
     pub window_properties: Option<WindowProperties>,
+    pub marks: Option<Vec<String>>,
```
Comment (JayceFayne) [inline src/reply.rs:218]:
```suggestion
	#[serde(default)]
    pub marks: Vec<String>,
```

**Comment by JayceFayne** [comment]
> By the way I've tested this and it works fine ok_hand

Yeah, lgtm! I added one suggestion ;)

**Comment by ammgws** [comment]
I'm on mobile now so could you please squash the commits for me?

**Comment by JayceFayne** [comment]
Ofc, np!

**Pull request merged**